release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -473,6 +473,127 @@ The Local Storage Operator now has the ability to:
 [id="ocp-4-6-operators"]
 === Operators
 
+[id="ocp-4-6-operator-api"]
+==== Operator API now supported
+
+The Operator API introduced in {product-title} 4.5 as a Technology Preview
+feature is now supported and enabled by default. Previously, installing
+Operators using Operator Lifecycle Manager (OLM) required cluster administrators
+to be aware of multiple APIs, including CatalogSources, Subscriptions,
+ClusterServiceVersions, and InstallPlans. This single Operator API resource is a
+first step towards a more simplified experience discovering and managing the
+lifecycle of Operators in a {product-title} cluster.
+
+While previously only available in read-only mode, cluster administrators can
+now use the web console, in addition to the CLI, with this single API to
```

Comment:
@adellape I do not think cluster administrator can use the API with web console, and only the CLI. @njhale could you please confirm it? thanks

Reply:
AFAIK the Operator API is not directly used by the console. I think @spadgett will be able to directly confirm/deny.

Reply:
The operator API isn't used directly by console. It is possible to view any k8s resource and edit its YAML, however, so you could use it that way. I'd still probably drop "web console" from this sentence.